src/api/public/apidocs-new/paths/worker_cmd_checkconstraints.yaml

```diff
@@ -0,0 +1,114 @@
+post:
+  summary: Lists workers wich match a constraints filter.
+  description: |
+    Given a project, package, repository and architecture, list workers wich can build with that restrictions, and also match a constraints filter.
+
+    This endpoint doesn't create or modify any data.
+
+    More information about constraints can be found in the
+    [user guide](https://openbuildservice.org/help/manuals/obs-user-guide/cha.obs.build_job_constraints.html).
+
+    This operation is the same as `POST /worker/command/run?cmd=checkconstraints`.
+  security:
+    - basic_authentication: []
+  parameters:
+    - in: query
+      name: project
+      schema:
+        type: string
+      required: true
+      example: home:user1
+      description: Project name.
+    - in: query
+      name: repository
+      schema:
+        type: string
+      required: true
+      description: Repository name.
+      example: openSUSE_Tumbleweed
+    - in: query
+      name: arch
+      schema:
+        type: string
+      required: true
+      description: Architecture name.
+      example: x86_64
```

Comment:
We could reuse the existing architecture with `- $ref: '../components/parameters/architecture_name.yaml'`

Reply:
That existing architecture is a path parameter. In this case this is a query parameter.